solutions/cf-1528A.mdx

```diff
@@ -0,0 +1,134 @@
+---
+id: cf-1528A
+source: CF
+title: Parsa's Humongous Tree
+author: Ryan Chou
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/91058)
+
+## Explanation
+
+Without loss of generality, let's assume we've already computed the best beauty values for all of vertex $u$'s neighbors.
+
+Let $\texttt{range}_u$ store $l_u$ and $r_u$ and $\texttt{adj}_u$ store the beauty values of all vertices adjacent to $u$.
+
+$\texttt{range}_u = \{1, 10\}$
```

Comment:
I think you should note that this an example

Reply:
this